src/odin/server/status/jobs_info.rs

```diff
@@ -8,15 +9,17 @@ pub struct JobInfo {
   pub schedule: String,
 }
 
-impl From<&str> for JobInfo {
-  fn from(job_name: &str) -> Self {
+impl FromStr for JobInfo {
+  type Err = ();
+
+  fn from_str(job_name: &str) -> Result<JobInfo, ()> {
```

Comment:
If an error can never occur then it would be [`std::convert::Infallible`](https://doc.rust-lang.org/std/convert/enum.Infallible.html)

Reply:
okidoki